Add release workflow

sdclarke opened the pull request:
Add a workflow which runs when new tags are created to push the role to Ansible Galaxy

sdclarke committed:
Add release workflow

This workflow will import the role into Ansible Galaxy when a new tag is
pushed to the repository

jjardon commented on .github/workflows/release.yaml:
```diff
@@ -0,0 +1,23 @@
+          ansible-galaxy role import
+          --token ${{ secrets.ANSIBLE_GALAXY_API_TOKEN }}
+          $(echo ${{ github.repository }} | cut -d/ -f1)
+          $(echo ${{ github.repository }} | cut -d/ -f2)
```
@sdclarke This looks good to me

Let's add a CI job to check the ansible correctness (so it runs when a PR is sent) (something like `ansible-playbook --syntax-check` or similar)

sdclarke replied:
Added a job to run ansible-lint on pull requests.

jjardon reviewed (changes requested):
Split the "Add linting to CI and fix linting failures" commit in 2 please: one to fix the problems, other to add the linting job

sdclarke committed:
Fix linting issues

The issues addressed are:
- lines which were too long
- truthy values not being specified with `true`
- adding the namespace to the metadata for the role

sdclarke committed:
Add workflow to perform linting on pull requests

sdclarke commented:
> Split the "Add linting to CI and fix linting failures" commit in 2 please: one to fix the problems, other to add the linting job

Done
